src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    private Colors color;
+    private double side;
```

Comment:
Think about which variables should be local in the method and which should be class-level. Not all figures might have a concept of 'side', so this might not be applicable to all figures.